src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -3,6 +3,7 @@
 
```

Comment:
@vexx32, your last commit had 3 failures in `PowerShell-CI-linux`
Test-Connection.Ping.Default parameter set is 'Ping'
```powershell
Expected the value to have type [System.Net.NetworkInformation.PingOptions] or any of its subtypes, but got $null with type $null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 45
45:             $result.Options | Should -BeOfType "System.Net.NetworkInformation.PingOptions"
```
Test-Connection.Ping.Repeat works
```powershell
Expected exactly '127.0.0.1', but got System.Object&, System.Private.CoreLib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e Address(int ).
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 205
205:             $pingResults.Address[0] | Should -BeExactly $targetAddress
```
Test-Connection.MTUSizeDetect.MTUSizeDetect works
```powershell
Expected strings to be the same, but they were different.
String lengths are both 8.
Strings differ at index 0.
Expected: 'fv-az644'
But was:  '10.1.0.4'
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 219
219:             $result.Destination | Should -BeExactly $hostName
```